docs/resource-policy/installation.md

```diff
@@ -103,6 +103,26 @@ along with the default values, for the Topology-aware and Balloons plugins Helm
 | `hostPort`         | 8891                                                                                                                          | metrics port to expose on the host                   |
 | `config`           | <pre><code>ReservedResources:</code><br><code>  cpu: 750m</code></pre>                                                        | plugin configuration data                            |
 
+#### Memtierd
+
+| Name               | Default                                                                                                                       | Description                                          |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| `image.name`       | [ghcr.io/containers/nri-plugins/nri-resource-policy-memtierd](ghcr.io/containers/nri-plugins/nri-resource-policy-balloons)    | container image name                                 |
+| `image.tag`        | unstable                                                                                                                      | container image tag                                  |
+| `image.pullPolicy` | Always                                                                                                                        | image pull policy                                    |
+| `resources.cpu`    | 250m                                                                                                                          | cpu resources for the Pod                            |
+| `resources.memory` | 100Mi                                                                                                                         | memory qouta for the                                 |
+
+#### Memory-qos
+
+| Name               | Default                                                                                                                       | Description                                          |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| `image.name`       | [ghcr.io/containers/nri-plugins/nri-resource-policy-memory-qos](ghcr.io/containers/nri-plugins/nri-resource-policy-balloons)  | container image name                                 |
```

Comment:
same here. it is nri memory management plugin, but not resource policy.

Reply:
Aha good to know. I will fix it here and in other locations.

Reply:
Coo, replacing every "`resource-policy-`" with empty string should do the trick.

Reply:
@kad TA and Balloons are classified as nri-resource-management. 
Are you suggesting to have another group(and directory) called something like nri-memory-management?